nibabel/freesurfer/mghformat.py

```diff
@@ -46,13 +46,13 @@
 # caveat 2: Note that the bytespervox you get is in str ( not an int)
 _dtdefs = (  # code, conversion function, dtype, bytes per voxel
     (0, 'uint8', '>u1', '1', 'MRI_UCHAR', np.uint8, np.dtype(np.uint8),
```

Comment:
Hum - this seems a bit harder to read, because it's difficult to distinguish new lines from line continuations.